source/uwp/SharedVisualizer/ViewModel/KeyboardPressTimeCounter.cs

```diff
@@ -56,7 +61,12 @@ public void CheckIfSendIsNecessary(Object stateInfo)
 
         private async Task SendShowErrorsUpdateAsync()
         {
+#if USE_WINUI3
+            var queueController = Microsoft.UI.Dispatching.DispatcherQueueController.CreateOnDedicatedThread();
+            await queueController.DispatcherQueue.EnqueueAsync(() =>
```

Comment:
Is there any specific reason you're not using
```
 var dispatcher = Microsoft.UI.Dispatching.DispatcherQueue.GetForCurrentThread();
 dispatcher.TryEnqueue(..):
```

Reply:
That thread doesn't have a dispatcher, so `Microsoft.UI.Dispatching.DispatcherQueue.GetForCurrentThread()` returns null. 
I probably should do `DispatcherQueueController.CreateOnCurrentThread()`, but still `TryEnqueue()` returns a bool, not something awaitable. So I used DispatcherQueueExtensions to match the existing functionality. Up to you if you'd rather not use the extension, but also not await. I'm pushing a change I think is better than what I have now.

Reply:
I don't mind either way :)
If there's truly a need to enqueue it `async`, then usage of extension is correct:)